stan/math/prim/mat/fun/abs.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_ABS_HPP
+#define STAN_MATH_PRIM_MAT_FUN_ABS_HPP
+
+#include <stan/math/prim/mat/vectorize/apply_scalar_unary.hpp>
+#include <cmath>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Structure to wrap abs() so it can be vectorized.
+     * @param x Argument variable.
+     * @tparam T Argument type.
+     * @return Absolute value of x.
+     */
+    struct abs_fun {
+      template <typename T>
+      static inline T fun(const T& x) {
+        using std::fabs;
```

Comment:
That include isn't doing anything.  But I don't think there is a `std::abs`.  If this is trying to wrap `abs()` as the doc says, you want `using stan::math::abs`.